types/ext/translation-internal.d.ts

```diff
@@ -47,3 +48,7 @@ export type DatabaseDeinflection = {
     inflectionRuleChainCandidates: Dictionary.InflectionRuleChainCandidate[];
     databaseEntries: DictionaryDatabase.TermEntry[];
 };
+
+export type TextTransformation = Language.TextTransformation & {
```

Comment:
Might be more clear if name is changed slightly, `TextTransformationWithSettingValue` or something like that? Makes it kinda verbose I guess. Alternatively, should this just be a new object that is `{transform: TextTransformation, setting: string}`?

Reply:
Changed to the latter.